e2e-test/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "test",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "lint": "eslint .",
```

Comment:
Have you tried running `yarn run lint`? Since this was just added it won't get called automatically in CI, so there's no way to know that this new code is actually linted properly.

Reply:
I will run `yarn run lint` and fix lint issues